### ISSUE TITLE
Error importing module dbatools

### ISSUE THREAD

**Issue by Zhunya** [sqlcollaborative/dbatools#1491]
## Is this:
_Just place "x" to indicate selection (e.g. "[x]")_

- [ ] *feature* 
- [x] *bug*:

## System Details

- Operating system name and version:
```
1703, 15063.413
```
- Output from `$PSVersionTable`:

```
Name                           Value
----                           -----
PSVersion                      5.1.15063.413
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.15063.413
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

- Output of dbatools version:

```
Latest available development tree
```
![default](https://user-images.githubusercontent.com/14327077/27576436-a5f30386-5b25-11e7-84d7-f3d75b50daca.png)

![default](https://user-images.githubusercontent.com/14327077/27576766-d58c1906-5b26-11e7-866d-afc262cb0068.png)

## $ERROR
[error.zip](https://github.com/sqlcollaborative/dbatools/files/1104627/error.zip)

## .Net Version
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\TeamExplorer>clrver

Microsoft (R) .NET CLR Version Tool  Version 4.0.30319.33440
Copyright (c) Microsoft Corporation.  All rights reserved.

Versions installed on the machine:
v2.0.50727
v4.0.30319
```
![default](https://user-images.githubusercontent.com/14327077/27577277-ba372ef0-5b28-11e7-9d27-f93faec73198.png)

**Comment by JohnRoos** [comment]
I noticed the same problem. The import works if I run the import-module command directly from the dbatools folder instead, if that would be of any help.
`C:\GIT\dbatools\> Import-Module C:\GIT\dbatools`

**Comment by Zhunya** [comment]
I tried JohnRoos trick, and it worked. dbatools.dll created once after first run. Every other module imports worked very good from any place. Have no time to run debug session, sorry.

**Comment by FriedrichWeinmann** [comment]
I've got some thoughts on this: It may be necessary to specify the build path explicitly when building the library during module import - I think the default configuration fails spectacularly, because the configuration in the project is understood as relative to the current directory, rather than the solution root.
Something like that or similar.
Either way, just returned from a night with the colleagues at the local pub so ... not the best time to troubleshoot stuff. Will look into it soonest tomorrow and push a fix.

The library is not pre-compiled in the dev repo on purpose, due to my inherent sadism and desire to avoid a bottleneck due to PR conflicts. It should be native part of master and gallery releases. Development should build on first local import. As soon as I've got the current issue fixed, that should work well enough for every-day usage.
